test/Microsoft.Extensions.Logging.Test/ConsoleLoggerTest.cs

```diff
@@ -923,6 +923,19 @@ public void ConsoleLoggerOptions_IncludeScopes_IsAppliedToLoggers()
             {LogLevel.Trace, "trce"},
         };
 
+        public static TheoryData<LogLevel> Levels
+        {
+            get
+            {
+                var data = new TheoryData<LogLevel>();
+                foreach (LogLevel value in Enum.GetValues(typeof(LogLevel)))
```

Comment:
In this case, implicit variable typing fails because Enum.GetValues returns `System.Array`.